amgut/handlers/auth_handlers.py

```diff
@@ -23,6 +23,8 @@ def get(self):
     @authenticated
     def post(self):
         skid = self.current_user
+        # log user out for register process
+        self.clear_cookie("skid")
```

Comment:
Would it be better to just disable the button as this might open the door to a race condition, where the user's kit is being registered, they get frustrated, and log back in themselves, meaning they either get back to the registration window (if they beat the adding the handout kit) or they get logged in successfully. The former would be a bad situation to encounter